edx_repo_tools/dev/show_hooks.py

```diff
@@ -7,7 +7,7 @@
 from git.repo.base import Repo
 
 from edx_repo_tools.auth import pass_github
-from helpers import paginated_get
+from edx_repo_tools.helpers import paginated_get
```

Comment:
I'm not following this one -- where is this module declared, if not at the top level?

Reply:
It's in the edx_repo_tools directory now, it had been at the top of the repo, which wouldn't work when installed.

Reply:
Oh, I see the rename now. :-)